app/controllers/api/v1/authors_controller.rb

```diff
@@ -0,0 +1,54 @@
+class Api::V1::AuthorsController < ApplicationController
+  before_action :load_author, only: %i(show update destroy)
+
+  def index
+    authors = Author.order_name
+    @authors = ActiveModelSerializers::SerializableResource.new(authors, {})
```

Comment:
```suggestion
    @authors = ActiveModelSerializers::SerializableResource.new(authors)
```
https://www.rubydoc.info/gems/active_model_serializers/ActiveModelSerializers/SerializableResource

Reply:
dạ, e đã fix